tests/pipelines/controlnet/test_controlnet_inpaint_sdxl.py

```diff
@@ -289,9 +298,7 @@ def test_controlnet_sdxl_guess(self):
 
         output = sd_pipe(**inputs)
         image_slice = output.images[0, -3:, -3:, -1]
-        expected_slice = np.array(
-            [0.5381963, 0.4836803, 0.45821992, 0.5577731, 0.51210403, 0.4794795, 0.59282357, 0.5647199, 0.43100584]
-        )
+        expected_slice = np.array([0.549, 0.5053, 0.4676, 0.5816, 0.5364, 0.483, 0.5937, 0.5719, 0.4318])
```

Comment:
The reason why this needs changing is because we're fixing the seeds when initializing the model level components of the pipeline.